composer.json

```diff
@@ -14,7 +14,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "psr/log": "1.0.0"
```

Comment:
This brings in https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md